### PR TITLE
#847 Stocktake operations

### PR DESCRIPTION
**Pull request by joshxg**
#847

- Just adds the queries with the correct params/field names for the API to an operations folder in the stocktake directory.

### CODE REVIEW

File: packages/common/src/operations.generated.ts [hunk 1]
```diff
@@ -4,70 +4,6 @@ import { GraphQLClient } from 'graphql-request';
 import * as Dom from 'graphql-request/dist/types.dom';
 import gql from 'graphql-tag';
 import { graphql, ResponseResolver, GraphQLRequest, GraphQLContext } from 'msw'
-export type StocktakeQueryVariables = Types.Exact<{
```
Comment (joshxg) [inline packages/common/src/operations.generated.ts:7]: removed the stocktake operations from common, breaking things. 1 step back two steps forward..?

File: packages/inventory/src/Stocktake/api/operations.graphql [hunk 2]
```diff
@@ -0,0 +1,128 @@
+fragment StocktakeRow on StocktakeNode {
```
Comment (joshxg) [inline packages/inventory/src/Stocktake/api/operations.graphql:1]: Adding the queries with correct params/fields etc

File: packages/inventory/src/Stocktake/api/api.ts [hunk 3]
```diff
@@ -3,48 +3,28 @@ import {
...
   InsertStocktakeLineInput,
   UpdateStocktakeLineInput,
-  ConnectorError,
-  OmSupplyApi,
-  StocktakeQuery,
-  StocktakeLineNode,
-  StocktakeLineConnector,
   UpdateStocktakeInput,
   RecordPatch,
+  SortBy,
```
Comment (joshxg) [inline packages/inventory/src/Stocktake/api/api.ts:8]: pretty much exactly like requisitions. Api file and hooks file updates..

**Comment by joshxg** [comment]
This is the messy stage - next is to run through the list view / detail view and actually utilise all this.

File: packages/inventory/src/Stocktake/api/api.ts (hunk 3 above)
Comment (mark-prins) [reply packages/inventory/src/Stocktake/api/api.ts:8]: and removal of the Guards 😁

**Review by mark-prins — APPROVED** [review]
🎉  looking good thanks!

**Pull request merged**